lib/ansible/modules/network/ovs/openvswitch_bridge.py

```diff
@@ -140,6 +140,12 @@ def map_obj_to_commands(want, have, module):
                                 or want['external_ids'][k] != have['external_ids'][k]):
                             command += " " + k + " " + v
                             commands.append(command)
+
+            if want['vlan'] != have['vlan']:
```

Comment:
This is where it is failing
```
cmd: /usr/bin/ovs-vsctl -t 5 set port br-test tag=None
msg: 'ovs-vsctl: "None" is not a valid integer or range'
```
You will need a similar condition like `if want['vlan'] and want['vlan'] != have['vlan']:`